checks/uenv/check_osu_micro_benchmarks.py

```diff
@@ -0,0 +1,123 @@
+# Copyright 2016-2023 Swiss National Supercomputing Centre (CSCS/ETH Zurich)
+# ReFrame Project Developers. See the top-level LICENSE file for details.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+class BaseCheck(rfm.RunOnlyRegressionTest):
+    valid_systems = ['*']
+    valid_prog_environs = ['+osu-micro-benchmarks']
+    sourcesdir = None
+    num_tasks = 2
+    num_tasks_per_node = 1
+    modules = ['osu-micro-benchmarks']
+    pmi = variable(str, value='')
+    env_vars = {
+        'MPIR_CVAR_ENABLE_GPU': 0,
+        'MPICH_GPU_SUPPORT_ENABLED': 0,
+        # Set to one byte more than the last entry of the test
+        'MPIR_CVAR_CH4_OFI_MULTI_NIC_STRIPING_THRESHOLD': 4194305
+    }
+    tags = {'uenv'}
+
+    @run_after('init')
+    def set_descr(self):
+        self.descr = f'{self.name} on {self.num_tasks} nodes(s)'
+
+    @run_after('setup')
+    def set_launcher_options(self):
+        if self.pmi != '':
+            self.job.launcher.options = [f'--mpi={self.pmi}']
```

Comment:
```suggestion
            self.job.launcher.options = \[self.current_environ.extras['launcher_options']]
```
as in https://github.com/eth-cscs/cscs-reframe-tests/blob/alps/config/systems/hohgant.py#L191